roles/yoda_portal/tasks/main.yml

```diff
@@ -105,6 +105,17 @@
   when: not ansible_check_mode
 
 
+- name: Ensure the portal config directory exists
+  ansible.builtin.file:
+    path: "{{ config_folder_path }}"
+    state: directory
+    owner: "{{ yoda_deployment_user }}"
+    group: "{{ yoda_deployment_user }}"
+    mode: '0775'
+  become: true
+  become_user: "root"
+
```

Comment:
I added a ansible.file task for creating the directory. Is it okay? I gave 775 permission because other folders in the same directory used this permission setting.

Reply:
Yeah, but it should be created by yoda_deployment_user. See other tasks here for examples.